uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Mail/BaseRequestNotificationTemplate.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+use CRM_HRLeaveAndAbsences_BAO_LeaveRequest as LeaveRequest;
+use CRM_HRLeaveAndAbsences_Service_LeaveRequestComment as LeaveRequestCommentService;
+
+abstract class CRM_HRLeaveAndAbsences_Mail_BaseRequestNotificationTemplate {
```

Comment:
Just to keep things a bit more organized, can we move the templates to their own folder? That is, rename them to: `CRM_HRLeaveAndAbsences_Mail_Template_<Name>`.

Example: `CRM_HRLeaveAndAbsences_Mail_Template_BaseRequestNotification`